### PR TITLE
Refactor test code

### PR DESCRIPTION
**Pull request by ybiquitous**
> Explain a summary, purpose, or background for this change.

- Unify format: `require "test_helper"`.
- Remove the `rr` gem. Instead, use `minitest/mock`.

> Link related issues or pull requests, e.g. `Fix #123`, `Related to #456`, or `None`.

None.

### CODE REVIEW

File: lib/runners/workspace.rb [hunk 1]
```diff
@@ -2,8 +2,6 @@ module Runners
   class Workspace
     include Tmpdir
 
-    class DownloadError < SystemError; end
-
```
Comment (ybiquitous) [inline lib/runners/workspace.rb:6]: [note] Unused.

File: test/options_test.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+require 'test_helper'
 
 class OptionsTest < Minitest::Test
   include TestHelper
 
-  def setup
-    stub(Runners::IO::AwsS3).stub? { true }
+  def run
+    Runners::IO::AwsS3.stub(:stub?, true) { super }
   end
```
Comment (ybiquitous) [inline test/options_test.rb:8]: [note] Override `Minitest::Test#run`.

**Review by yykamei — COMMENTED** [review]
LGTM

Please explain why you did these tasks :pray: 

> * Unify format: require "test_helper".
> * Remove the rr gem. Instead, use minitest/mock.

**Comment by ybiquitous** [comment]
> - Unify format: require "test_helper".

If we are using `require_relative "../test_helper"`, we need to consider a test file location and a relative path to `test_helper.rb`. It is not useful.

> - Remove the `rr` gem. Instead, use `minitest/mock`.

I think that using multiple mock libraries leads to the source of confusion and high learning cost.
Minitest is very popular (de facto standard), while RR is not.

**Review by yykamei — APPROVED** [review]

**Comment by ybiquitous** [comment]
Thanks for the review! 👍

**Pull request merged**